ros/src/waypoint_follower/src/pure_pursuit_core.cpp

```diff
@@ -252,7 +252,8 @@ bool PurePursuit::verifyFollowing() const
 geometry_msgs::Twist PurePursuit::calcTwist(double curvature, double cmd_velocity) const
 {
   // verify whether vehicle is following the path
-  bool following_flag = verifyFollowing();
+  // bool following_flag = verifyFollowing();
+  bool following_flag = false;
```

Comment:
I should have added a comment for sure.
This forces the `waypoint_follower` to always update angular velocity.
The change makes the car steering more accurate.

Probably a better way would be to modify `verifyFollowing` so make it more responsive.
I think it can be done later during optimization phase (if we have time)

Reply:
Created a card for this: https://trello.com/c/Wq5gR1gA/18-make-the-verifyfollowing-method-of-waypointfollowers-more-responsive